src/store/channels-list/saga.test.ts

```diff
@@ -186,34 +134,14 @@ describe('channels list saga', () => {
       groupChannelType: '',
     };
 
-    const { storeState } = await expectSaga(fetchChannelsAndConversations)
-      .provide([
-        [
-          matchers.call(chat.get),
-          chatClient,
-        ],
-        [
-          matchers.call(
-            [
-              chatClient,
-              chatClient.getChannels,
-            ],
-            rootDomainId
-          ),
-          [channel],
-        ],
-        [
-          matchers.call([
-            chatClient,
-            chatClient.getConversations,
-          ]),
-          [conversation],
-        ],
-        [
-          matchers.call.fn(delay),
-          null,
-        ],
-      ])
+    const additionalProviders = [
+      [matchers.call(chat.get), chatClient],
```

Comment:
Shouldn't need this line in any of the tests now that it's defaulted in `subject`